mslib/index.py

```diff
@@ -103,15 +107,17 @@ def get_topmenu():
             (url_for('index'), 'Mission Support System',
              ((url_for('about'), 'About'),
               (url_for('install'), 'Install'),
+              (url_for("plots"), 'Gallery'),
```

Comment:
we share index.py for mscolab too. This now has an empty page stating 

"Gallery was not generated for this server."

We should extend this with a link to mss.rtfd.io/gallery/index.html

An example can be seen on http://mss.rtfd.io/gallery/index.html  To build this overview see the examples on ... 

Or on mscolab server we don't show that menu entry, this would be cleaner.

Reply:
Done